Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -130,23 +130,37 @@ open class PassiveLocationManager: NSObject {
     }
 
     /**
-     Pauses the Free Drive session.
+     Stops the Free Drive session associated with this instance.
+
+     Stopped trip sessions don't generate location and navigation status updates.
 
      Use this method to extend the existing Free Drive session if you temporarily don't need navigation updates. For
-     more info, read the [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
+     For more info, read the [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
      */
-    public func pauseTripSession() {
-        BillingHandler.shared.pauseBillingSession(with: sessionUUID)
+    public func stopTripSession() {
```

Comment:
`startUpdatingLocation()`, `startUpdatingHeading()`, `stopUpdatingLocation()`, and `stopUpdatingHeading()` use the verbs “start” and “stop” because they start and stop a series of updates and there’s no meaningful difference between starting and resuming that stream of updates. By contrast, a “session” is a meaningful mode of operation; a developer would hesitate to call `startTripSession()` multiple times because they would be billed for multiple sessions. However, these methods try to reuse an existing session, so “pause” and “resume” would be more accurate verbs to use in these method names.